Makefile

```diff
@@ -113,7 +113,7 @@ test-unit: $(SETUP_ENVTEST) ## Run the unit tests
 e2e: KIND_CLUSTER_NAME=operator-controller-e2e
 e2e: KUSTOMIZE_BUILD_DIR=config/e2e
 e2e: GO_BUILD_FLAGS=-cover
-e2e: run kind-load-test-artifacts test-e2e e2e-coverage kind-cluster-cleanup ## Run e2e test suite on local kind cluster
+e2e: kind-cluster-cleanup run kind-load-test-artifacts test-e2e e2e-coverage kind-cluster-cleanup ## Run e2e test suite on local kind cluster
```

Comment:
This is a preemptive cleanup of a pre-existing e2e cluster. I was running into the slight annoyance of the e2e failing, the last `kind-cluster-cleanup` not running, and then having to manually delete the cluster to be able to run the e2e again.

Reply:
I see @tmshort has a more wide-ranging fix in #353, so removing the Makefile changes from this PR.

Reply:
Adding `kind-cluster-cleanup` early in the dependency line won't work; as `make` will have recognized that target as already completed by the time it gets to the second `kind-cluster-cleanup`. So it won't be run, and it will never clean up!